filament/include/filament/Box.h

```diff
@@ -158,6 +158,23 @@ struct Aabb {
     bool isEmpty() const noexcept {
         return any(greaterThanEqual(min, max));
     }
+
+    struct Corners {
+        using value_type = math::float3;
+        value_type const* begin() const { return vertices; }
```

Comment:
Are you planning to use these iterator methods somewhere?

Reply:
yeah, it's used in the ShadowMap code for `for`